python/pyspark/ml/wrapper.py

```diff
@@ -82,13 +71,16 @@ def _transfer_params_to_java(self):
                 pair = self._make_java_param_pair(param, paramMap[param])
                 self._java_obj.set(pair)
 
-    def _transfer_params_from_java(self):
+    def _transfer_params_from_java(self, withParent=False):
```

Comment:
I think it'd be better to set the Param.parent field for each Param in load().  I don't think we'd reuse this functionality elsewhere, so it would be good to isolate to load().